packages/canary/tailwind.config.js

```diff
@@ -57,12 +59,27 @@ module.exports = {
           DEFAULT: 'hsl(var(--card))',
           foreground: 'hsl(var(--card-foreground))'
         },
+        'navbar-icon': {
+          secondary: 'hsl(var(--navbar-icon-secondary))'
+        },
+        'navbar-text': {
+          secondary: 'hsl(var(--navbar-text-secondary))'
+        },
         white: {
           DEFAULT: 'hsl(var(--white))'
         },
         black: {
           DEFAULT: 'hsl(var(--black))'
         },
+        red: {
+          DEFAULT: 'hsl(var(--red))'
+        },
+        green: {
+          DEFAULT: 'hsl(var(--green))'
+        },
+        orange: {
+          DEFAULT: 'hsl(var(--orange))'
+        },
```

Comment:
we had agreed to only add semantic token in tailwind.. adding `red`, `green`, `orange` etc goes against that. please remove and use the `destructive`, `success` and `warning` tokens instead